packages/runtime/src/widgets/Form/Dropdown.tsx

```diff
@@ -150,7 +168,10 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
             value={value}
           >
             <CustomScopeProvider value={item as CustomScope}>
-              {EnsembleRuntime.render([itemTemplate.template])}
+              {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+              <div onClick={(e) => e.stopPropagation()}>
+                {EnsembleRuntime.render([itemTemplate.template])}
+              </div>
```

Comment:
Make this a re-usable function and only add the wrapper element if manual close is true.